public/data/shivanshyadav.json

```
@@ -14,6 +14,11 @@
       "url": "https://twitter.com/_shivansh_13",
       "icon": "twitter"
     },
+    {
+      "name": "Connect with me on Linkedin",
```

The `i` isn't capitalized here. What do you think if we capitalize it? :)
```suggestion
      "name": "Connect with me on LinkedIn",
```